addons/mail/static/src/core/common/attachment_list.js

```diff
@@ -102,4 +102,9 @@ export class AttachmentList extends Component {
     get isInChatWindowAndIsAlignedLeft() {
         return this.env.inChatWindow && !this.env.alignedRight;
     }
+
+    get showDeleteOnAttachmentCard() {
+        return this.attachment.isDeletable && (!this.env.message || this.env.message?.hasTextContent)
+            || this.env.inComposer || this.props.attachments.length > 1
+    }
```

Comment:
Don't forget to enable JS tooling (see `web/tooling/enable.sh`), so that diff uses prettier.
This makes code more readable, easing review process.